Ftp/Rename-FtpDirectory/Rename-FtpDirectory.psm1

```diff
@@ -14,34 +14,34 @@ function Rename-FtpDirectory
     (
         [Parameter(Mandatory = $true,
             ValueFromPipelineByPropertyName = $true,
-            HelpMessage = "Specify a valid FTP server path to a folder that contains the directory which
-                   needs to be renamed.")]
+            HelpMessage = 'Specify a valid FTP server path to a folder that contains the directory which
+                   needs to be renamed.')]
         [string] $Url,
 
         [Parameter(Mandatory = $true,
             ValueFromPipelineByPropertyName = $true,
-            HelpMessage = "Provide username.")]
+            HelpMessage = 'Provide username.')]
         [string] $User,
 
         [Parameter(Mandatory = $true,
             ValueFromPipelineByPropertyName = $true,
-            HelpMessage = "Provide password in SecureString format.")]
+            HelpMessage = 'Provide password in SecureString format.')]
         [securestring] $Password,
 
         [Parameter(Mandatory = $true,
-            HelpMessage = "Specify folder to rename.")]
+            HelpMessage = 'Specify folder to rename.')]
         [string] $SourceFolder,
 
         [Parameter(Mandatory = $true,
-            HelpMessage = "Specify new folder name.")]
+            HelpMessage = 'Specify new folder name.')]
         [string] $DestinationFolder
     )
 
     Process
     {
-        $folderToRenamePath = $Url + "/" + $SourceFolder
-        $ftpFolderPath = $Url + "/" + $DestinationFolder
-        $destinationFolderRelPath = "../" + $DestinationFolder
+        $folderToRenamePath = $Url + '/' + $SourceFolder
+        $ftpFolderPath = $Url + '/' + $DestinationFolder
+        $destinationFolderRelPath = '../' + $DestinationFolder
```

Comment:
Why not keep the quotes and include the variables? e.g. `$folderToRenamePath = "$Url/$SourceFolder"`

Reply:
I didn't look into much refactoring (even at this scale), just fixing the immediate issues, but let's apply these changes, sure.